docs/product-and-design/analytics.md

```diff
@@ -0,0 +1,96 @@
+# Analytics
+
+The Cal-ITP Benefits application uses Amplitude to collect specific user properties and event data to analyze application usage.
+
+## Information not collected
+
+- IP Address
+- Designated Market Area (DMA)
+
+See the [Amplitude analytics code on GitHub](https://github.com/cal-itp/benefits/blob/dev/benefits/core/templates/core/includes/analytics.html#L30).
```

Comment:
@angela-tran Do you think I should add anything else here under this category? (Like we could specify we're not collecting e-mail addresses, SSNs, or any information on the Login.gov side, not collecting Agency card credentials)

Reply:
It doesn't seem necessary to me to maintain a list on this page of the external data that we don't collect. Our [project overview/security](https://docs.calitp.org/benefits/#data-storage) documentation has covered that we don't have access to any of the credentials used to verify eligibility, whether Login.gov or Agency card

Reply:
Definitely makes sense to call out that we specifically configure Amplitude to not collect IP and DMA 👍